spec/models/ip_spec.rb

```diff
@@ -50,4 +50,32 @@
       it { is_expected.to be_available }
     end
   end
+
+  context 'when many IPs already exist for the same location' do
+    let(:location_1) do
+      Location.create!(
+        address: 'Momentum Centre, London',
+        postcode: 'SE10SX',
+        organisation_id: create(:organisation).id
+      )
+    end
+
+    let(:ip) do
+      Ip.new(location_id: location_1.id, address: "127.0.2.1")
+    end
+
+    before { 20.times { create(:ip, location: location_1) } }
+
+    it 'saves without timing out' do
```

Comment:
Do we still need this given we have fixed the root cause of the issue?

Reply:
Yeah, I wasn't sure either. Let's chop it out, less code is good?